pkg/nsx/services/vpc/vpc.go

```diff
@@ -0,0 +1,59 @@
+package vpc
+
+import (
+	"sync"
+
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/logger"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+)
+
+var (
+	log                       = logger.Log
+	MarkedForDelete           = true
+	EnforceRevisionCheckParam = false
```

Comment:
can line #15 and #16 move to common? seems VPC, securitypolicies are all use it.

Reply:
I removed it since currently it is not used. but i do not think we should move them to common, this value is not a common static string value, it is used in logic and passed through different method via pointer, move it to common will cause the value be accidently changed by other services.